### PR TITLE
e2e swap fixes, bump chromedriver

### PR DESCRIPTION
**Pull request by BrodyHughes**
## What changed (plus any additional context for devs)
- E2e is currently unable to run on local
- Greg's PR that added popular in rainbow indirectly caused a couple of these issues
- If a token is in the 'popular in Rainbow' section, it doesn't appear in other sections of the search. `gmx` sometimes appears here and sometimes doesn't. When greg merged his PR it wasn't so e2e was passing but now it is, so it failed. 
- to fix this, added a couple try / catch blocks when we search for this specific token
- added a couple of delays in areas i was seeing random sporadic failures as well.
- also bumping chromedriver / chrome version to match my local so I can run tests

### CODE REVIEW

File: e2e/serial/swap/1_swapFlow1.test.ts [hunk 1]
```diff
@@ -764,27 +796,25 @@ it('should be able to find exact match on other networks', async () => {
...
-
-  // await findElementByTestIdAndClick({
-  //   id: `${SWAP_VARIABLES.OP_OPTIMISM_ID}-other_networks-token-to-buy-row`,
-  //   driver,
-  // });
-  // await findElementByTestIdAndClick({
-  //   id: `${SWAP_VARIABLES.OP_OPTIMISM_ID}-token-to-buy-token-input-remove`,
-  //   driver,
-  // });
+  await typeOnTextInput({
```
Comment (BrodyHughes) [inline e2e/serial/swap/1_swapFlow1.test.ts:799]: #1732 was merged

File: e2e/serial/swap/1_swapFlow1.test.ts [hunk 2]
```diff
@@ -108,6 +108,11 @@ it('should be able to connect to hardhat', async () => {
   const button = await findElementByText(driver, 'Disconnect from Hardhat');
   expect(button).toBeTruthy();
   await findElementByTestIdAndClick({ id: 'navbar-button-with-back', driver });
+
+  // sometimes hardhat balances take time to display.
+  // If we go straight to swap and they aren't updated,
+  // the swap gets auto-populated with the wrong token.
+  await delay(10_000);
```
Comment (BrodyHughes) [inline e2e/serial/swap/1_swapFlow1.test.ts:115]: was seeing failures maybe 10% of the time bc of this. Feels better to have it than not.

**Review by greg-schrammel — APPROVED** [review]
sorry 🙃

**Comment by BrodyHughes** [comment]
> sorry 🙃

tbh not really your fault. It passed when we merged your PR haha

**Review by DanielSinclair — APPROVED** [review]
Puhed up a policy change from `yarn setup` and that seem to fix the CI build issue. Looks good to me!

**Pull request merged**